internal/example/main.go

```diff
@@ -30,21 +30,45 @@ import (
 
 //nolint:all
 var (
```

Comment:
can we split this PR into 2, the first with the lint fixes and then changes to the lint CI ? or would it be a chicken and egg problem here?

Reply:
It would work, but we wouldn't be able to verify on the CI that the fixes will make the checks pass, so it's useful IMO to submit them together.

Reply:
The only changes that don't come from lint fixes are in `golangci.yml` and in `ci.yml`.